client/signup/steps/theme-selection/theme-thumbnail.jsx

```diff
@@ -7,14 +7,18 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var analytics = require( 'analytics' ),
-	SignupActions = require( 'lib/signup/actions' ),
-	ThemeHelper = require( 'lib/themes/helpers' );
+	SignupActions = require( 'lib/signup/actions' );
 
 module.exports = React.createClass( {
 	displayName: 'ThemeThumbnail',
 
+	propTypes: {
+		themeName: React.PropTypes.string.isRequired,
+		themeSlug: React.PropTypes.string.isRequired,
+	},
```

Comment:
Ooooh. I didn't know about `PropType.Shape`. I figured it's good to split them up so it's explicit what is required by this component, but `Shape` might change that. I'll look into it.